go/oasis-node/cmd/common/pprof/pprof.go

```diff
@@ -30,6 +35,33 @@ type pprofService struct {
 	errCh chan error
 }
 
+// DumpHeapToFile writes the current process heap to given file with unique suffix.
+func DumpHeapToFile(name string) error {
+	// Find unique filename.
```

Comment:
Instead of manually doing this use [TempFile](https://golang.org/pkg/io/ioutil/#TempFile)